root_locals.tf

```diff
@@ -19,7 +19,7 @@ locals {
   )
   database_availability_zones = ["eu-west-2a", "eu-west-2b"]
 
-  region                      = "eu-west-2"
+  region = "eu-west-2"
```

Comment:
Did you run terraform fmt?

Reply:
Yeah, I ran it on all files, this one wasn't formatted before.